src/Microsoft.AspNetCore.Sockets.Client.Http/ITransport.cs

```diff
@@ -9,8 +9,8 @@ namespace Microsoft.AspNetCore.Sockets.Client
 {
     public interface ITransport
     {
-        Task StartAsync(Uri url, IDuplexPipe application, TransferMode requestedTransferMode, IConnection connection);
+        Task StartAsync(Uri url, IDuplexPipe application, TransferFormat requestedTransferMode, IConnection connection);
         Task StopAsync();
-        TransferMode? Mode { get; }
+        TransferFormat? Format { get; }
```

Comment:
Does this exist only for websockets?

Reply:
.NET Client hasn't really been touched by this change yet. That's in the TODO list for today.